cli/src/commands/serve.ts

```diff
@@ -0,0 +1,49 @@
+import { Command, flags } from "@oclif/command";
+import { parse } from "../../../lib/src/neu/parser";
+import { runValidationServer } from "../../../lib/src/validation-server/server";
+
+const ARG_API = "spot_contract";
+
+/**
+ * oclif command to start the spot contract validation server
+ */
+export default class Serve extends Command {
```

Comment:
We have different types of Spot server, for example `spot mock` also starts a server. Ultimately I think we should merge them under different paths. In the meantime, I'd recommend something less general e.g. `spot validation-server`.